submodules/eks/iam.tf

```diff
@@ -191,11 +191,35 @@ data "aws_iam_policy_document" "snapshot" {
   }
 }
 
+data "aws_iam_policy_document" "ssm" {
+  statement {
+    effect    = "Allow"
+    resources = ["*"]
+
+    actions = [
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams"
+    ]
+  }
```

Comment:
Can we limit these permissions a bit more granularly?

https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-rc-setting-up-cwlogs.html

Reply:
Works for me!